File: Python/secao07/Exercicio05.py
# input
nome = input("Informe seu nome: ")
senha = input("Informe uma senha: ")

# processing
while nome == senha:
    print("O nome não pode ser igual a senha.")
    nome = input("Informe novamente seu nome: ")
    senha = input("Informe novamente a senha: ")
print("===============================")
print("||    Usuário cadastrado.    ||")
print("===============================")

File: Java/src/secao04/Exercicio05.java
package secao07;

import java.util.Scanner;

public class Exercicio05 {
    public static void main(String[] args) {
        // var
        String nome, senha;
        Scanner teclado = new Scanner(System.in);

        // input
        System.out.println("Informe seu nome: ");
        nome = teclado.nextLine();
        System.out.println("Informe sua senha: ");
        senha = teclado.nextLine();
        
        // processing
        while (nome.equals(senha)) {
            System.out.println("ERRO: A senha e o nome de usuário são iguais.");
            System.out.println("======================================");
            System.out.println("Informe seu nome: ");
            nome = teclado.nextLine();
            System.out.println("Informe sua senha: ");
            nome = teclado.nextLine();
        }

        // output
        System.out.println("Cadastro realizado com sucesso.");
        teclado.close();
    }
}
